pom.xml

```diff
@@ -64,36 +64,36 @@
             <dependency>
                 <groupId>com.okta.sdk</groupId>
                 <artifactId>okta-sdk-api</artifactId>
-                <version>0.10.1-SNAPSHOT</version>
+                <version>0.11.0-SNAPSHOT</version>
```

Comment:
Seems a little repetitive in this file.

Reply:
Yeah, but using ${project.version} isn't ideal in some cases.  The versions get updated by the release plugin automatically and by `mvn versions:set` manually